assetsys.h

```diff
@@ -5782,8 +5867,11 @@ static void assetsys_internal_recurse_directories( assetsys_t* sys, int const co
             strcat( sys->temp, name );
 
             struct stat s;
+            assetsys_internal_time_t time;
             if( stat( sys->temp, &s ) == 0 )
                 {
+                ASSETSYS_ASSERT( assetsys_internal_filetime( sys->temp, &time.time ) );
```

Comment:
Hmm, asserts might be compiled out in release builds, leaving time uninitialized, right?

Reply:
Oh right, whoops! Thanks for the catch.